ochazuke/webhooks/helpers.py

```diff
@@ -0,0 +1,235 @@
+                is_open=True, created_at=info.get('created_at'),
+                milestone_id=milestone.id)
+    # TODO: log failures/errors as well?
+    # Add issue to staging
+    db.session.add(bug)
+    # Perform the actual insertion to the database
+    db.session.commit()
+    log = app.logger
+    log.setLevel(logging.INFO)
+    msg = 'New issue ({iss}) successfully added to database.'.format(iss=bug)
+    log.info(msg)
+
+
+def new_event(info):
+    """Create an event object to insert into db from a new issue event.
+
+    When a new event is signaled, we insert it into the event table, including:
+    - github issue number (int, 'issue_id')
+    - username of user who triggered the event (text, 'actor')
+    - what the event was (text, 'action')
+    - any relevant details (json, 'details') -- see models.Event
+    - when we received the event (timestamp, 'received_at')
+    We assign each event a unique id automatically upon insertion to the db.
+    """
+    event = Event(issue_id=info.get('issue_id'), actor=info.get('actor'),
+                  action=info.get('action'), details=info.get('details'),
+                  received_at=info.get('received_at'))
+    # TODO: log failures/errors as well?
+    # Add event to staging
+    db.session.add(event)
+    # Insert event into event table
+    db.session.commit()
+    log = app.logger
+    log.setLevel(logging.INFO)
+    msg = 'New event ({evt}) successfully added to database.'.format(evt=event)
+    log.info(msg)
+
+
+def issue_header_edit(info):
+    """Update issue table with edited header text."""
+    # Fetch existing issue from issue table
+    bug = Issue.query.get(info.get('issue_id'))
+    # Update header and commit changes
+    bug.header = info.get('header')
+    db.session.commit()
+
+
+def issue_status_change(info):
+    """Toggle an issue's 'is_open' status in table between true and false."""
+    bug = Issue.query.get(info.get('issue_id'))
+    bug.is_open = (False if bug.is_open else True)
+    db.session.commit()
+
+
+def issue_milestone_change(info):
+    """Add or remove an issue's milestone after an issue milestone event."""
+    issue = Issue.query.get(info.get('issue_id'))
+    if info.get('action') == 'milestoned':
+        issue.milestone = info.get('milestone_id')
+    else:
+        issue.milestone = None
+    db.session.commit()
+
+
+def issue_label_change(info):
+    """Add or remove an issue label after an issue label event."""
+    label_name = info.get('details')['label name']
+    label_id = Label.query.filter_by(name=label_name).one().id
+    issue = Issue.query.get(info.get('issue_id'))
+    if info.get('action') == 'labeled':
+        issue.labels.append(label_id)
+    else:
+        issue.labels.remove(label_id)
+    db.session.commit()
+
+
+def process_label_event_info(payload):
+    """Extract necessary information from webhook for label events."""
+    action = payload.get('action')
+    label_name = payload.get('label')['name']
```

Comment:
is there any moment where this is not true with the same reasoning than the previous comment.
```python
label_name = payload['label']['name']
```